### ISSUE TITLE
DM: Home page - where you're signed in - compatibility sessions

### ISSUE THREAD

**Issue by kerryarchibald** [matrix-org/matrix-authentication-service#1502]
Depends on #1499

compatibility sessions are displayed under browser sessions on the home page

AC:
- Subheader for compatibility sessions
- Latest 5 sessions are displayed
- A 'View all' button follows the list and navigates to `/account/compat-sessions`

**Comment by Johennes** [comment]
Superseded by https://github.com/matrix-org/matrix-authentication-service/issues/1546
